content/scalers/aws-kinesis.md

```diff
@@ -0,0 +1,99 @@
++++
+fragment = "content"
+weight = 100
+title = "AWS Kinesis Stream"
+background = "light"
++++
+
+Scale applications based on AWS Kinesis Stream.
+
+**Availability:** v1.0+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `aws-kinesis-stream` trigger that scales based on the shard count of AWS Kinesis Stream.
+
+```yaml
+triggers:
+  - type: aws-sqs-queue
+    metadata:
+      # Required: streamName
+      streamName: myKinesisStream
+      # Required: awsRegion
+      awsRegion: "eu-west-1"
+      # Optional: Default: 2
+      shardCount: "2"
+```
+
+**Parameter list:**
+
+- `streamName` - Kinesis Stream name
+- `shardCount` - The target value that a Kinesis data streams consumer can handle.
+- `awsRegion` - AWS Region for the Kinesis Stream
+
+### Authentication Parameters
+
+You can use `TriggerAuthentication` CRD to configure the authenticate by providing either a role ARN or a set of IAM credentials.
```

Comment:
TriggerAuth CRD is optional so we might want to clarify this here.

See https://keda.sh/concepts/authentication/

Reply:
I updated the description and added the link https://github.com/kedacore/keda-docs/pull/61/files#diff-664a100428eb94713fde5e3721428185R38
anything could be improved? thanks!

Reply:
Thanks!